extension/src/sidebar.css

```diff
@@ -104,25 +132,157 @@ a.promnesia-visited {
 #visits .datetime {
     white-space: nowrap;
     float: right;
-
-    color: #666;
-
     cursor: pointer;
+    font-family: monospace;
 }
 
-
 /* TODO not great, but quickest way to render it neatly */
 /* TODO not sure which order users expect first? */
 #visits .time {
-    /* color: blue; */
-    margin-right: 1em;
+    padding-right: 0.5em;
+    margin-right: 0.5em;
+    color: var(--time-color);
 }
+
 #visits .date {
-    /* color: red; */
     width: 11ch; /* TODO not sure how to alighn them properly across all of boxes... */
     display: inline-block; /* otherwise width doesn't work on span */
     text-align: end;
+    border-radius: 6px;
+    padding: 2px 4px;
+    color: var(--color2);
+    background: var(--date-bg);
+}
 
+#promnesia-sidebar {
+    display: flex;
+    flex-flow: column;
+    height: 100%;
+    margin: 0;
+    background: var(--bg);
+    overscroll-behavior: contain;
 }
 
+#promnesia-sidebar-header {
+}
 
+#promnesia-sidebar-toolbar {
+    padding: 6px 8px 0 8px;
+    border-bottom: 1px solid var(--bg);
+    overflow: hidden;
+    background: var(--header-bg);
+}
+
+#promnesia-sidebar-toolbar .button.active {
+    border-color: var(--source-color);
+}
+
+#promnesia-sidebar-toolbar .button-close {
+    float: right;
+    border-radius: 999px;
+    padding: 4px;
+}
+
+#promnesia-sidebar-container {
+    flex-grow: 1;
+    overflow-y: auto;
+}
+
+#promnesia-sidebar-filters {
+    text-transform: capitalize;
+    padding: 6px 10px 0px 10px;
+    background: linear-gradient(to bottom, var(--bg3) 0%, var(--bg) 100%);
```

Comment:
imo kinda subjective, so maybe keep in the private config? but don't mind it either

Reply:
ok

Reply:
i'll make it lighter, but it do visually beautify a lot )

Reply:
yeah it's fine to keep here, don't worry